cosmos/azure-spring-data-cosmos/spring3-samples/cosmos-aad-sample/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../../pom.xml</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>cosmos-spring-aad-sample</artifactId>
+  <version>1.0.0</version>
+
+  <properties>
+    <spring.boot.version>2.7.1</spring.boot.version>
```

Comment:
Why use sb2 in the sb3 sample?

Reply:
Not applicable with below comment. Line removed.